Functions.Templates/Bindings/bindings.json

```diff
@@ -1755,6 +1755,17 @@
             "enabledInTryMode": false,
             "documentation": "$content=Documentation\\Token.md",
             "filters": ["MSGraph"],
+            "AADPermissions": [
```

Comment:
For help text, you can have a "i" icon that shows a tool tip similar to what we have for other things.